phoenix-queryserver-it/pom.xml

```diff
@@ -213,5 +175,92 @@
       <artifactId>bcprov-jdk15</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
+
+  <profiles>
+    <!-- Phoenix 4.16.0-SNAPSHOT doesn't need special handling -->
+    <profile>
+      <id>phoenix-5.1.0-SNAPSHOT</id>
+      <activation>
+        <property>
+          <name>phoenix.version</name>
+          <value>5.1.0-SNAPSHOT</value>
+        </property>
+      </activation>
```

Comment:
Is there a reason to only have the activation for 5.1.0-SNAPSHOT?

Reply:
5.0.0 includes PQS, and 5.1.0 is not yet released.
I was planning to add 5.1.0 after it is released.
However, I am not against adding either or both to the activation criteria now.

Reply:
It just seemed strange to have activation for one of them but not the others. I think we should just be consistent in one direction :)